docs/guides/index.md

```diff
@@ -260,7 +262,7 @@ created by creating a sphere that wraps the scene:
 
 [next]: ./build-with-components.md
 
-And that is a very basic scene that places static objects in a 3D space using
-HTML. A good VR experience requires rich interaction and dynamic behavior. With
-the help of using and writing A-Frame components, we can [build a more advanced
+That is all it takes to create a very basic scene which places static objects in a 3D space using
+HTML! A good VR experience requires rich interaction and dynamic behavior. With
```

Comment:
The exclamation is finishing off the "That's all it takes" part; we're meaning to hype people up! :)